test/CountedPtrBenchmarks.h

```diff
@@ -46,10 +46,12 @@
 #include "MezzTest.h"
 
 #include "CountedPtr.h"
+#include "CountedPtrTests.h"
```

Comment:
@Sqeaky
Would including a test file inside a test file cause any problems? I don't recall ever doing this before. Would it create a problematic double declaration? Obviously it works, according to CI, but does it introduce other problems we're not seeing yet?

Reply:
It shouldn't, but it does feel wrong. It passes in CI so I am inclined to leave it.